src/librustdoc/json/conversions.rs

```diff
@@ -102,20 +105,22 @@ impl JsonRenderer<'_> {
     }
 }
 
+// FIXME: The trait names don't quite reflect reality anymore.
+
```

Comment:
I think it would be nicer to crate a new `Context` struct (like [SharedContext](https://doc.rust-lang.org/nightly/nightly-rustc/rustdoc/html/render/context/struct.SharedContext.html)) to put both the `Cache` and `tcx`, beacuse most of the time these are used their just calling more `from_tcx`, so I'd rather optimize to make calling quicer, and then the fields can be pulled out when needed. This also makes it easier if their's more shared state we need to add later.